release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -405,6 +405,17 @@ If legacy applications are sensitive to the capitalization of HTTP header names,
 
 {product-title} will update to HAProxy 2.2, which down-cases HTTP header names by default, for example, changing `Host: xyz.com` to `host: xyz.com`. Make sure to add the necessary configuration by using `spec.httpHeaders.headerNameCaseAdjustments` before upgrading {product-title} when HAProxy 2.2 is available.
 
+[id="ocp-4-7-networking-k8s-nmstate"]
+==== Kubernetes NMState Operator (Technology Preview)
+
+{product-title} 4.7 provides post-installation state-driven network configuration on the secondary network interfaces of cluster nodes using the Kubernetes NMState Operator as a Technology Preview feature. For more information, see
+xref:../networking/k8s_nmstate/k8s-nmstate-about-kubernetes-nmstate.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Using Kubernetes NMState (Technology Preview)].
```

Comment:
This should work: 
xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Using Kubernetes NMState (Technology Preview)].